drivers/gles2/shaders/scene.glsl

```diff
@@ -1562,9 +1559,10 @@ float sample_shadow(highp sampler2D shadow, highp vec4 spos) {
 
 #endif
 
-#if !defined(SHADOW_MODE_PCF_5) || !defined(SHADOW_MODE_PCF_13)
+#if !defined(SHADOW_MODE_PCF_5) && !defined(SHADOW_MODE_PCF_13)
 
-	return SAMPLE_SHADOW_TEXEL_PROJ(shadow, spos);
+	return SAMPLE_SHADOW_TEXEL(shadow, pos, depth);
+	
```

Comment:
```suggestion

```

Looks like there is some extra whitespace here which is causing CI to fail. You'll have to delete it and force push to get CI to pass. 

While you are at it, you will need to merge your commits together so that the PR only has one commit with all your changes. How to do so is described in detail in the docs here https://docs.godotengine.org/en/stable/community/contributing/pr_workflow.html#modifying-a-pull-request